Detections/SecurityEvent/RDP_RareConnection.yaml

```diff
@@ -65,27 +65,34 @@ query: |
   ) on Account, Computer
   | summarize StartTime = min(StartTime), EndTime = max(EndTime), ConnectionCount = sum(ConnectionCount)
   by Account, Computer, IpAddress, AccountType, Activity, LogonTypeName, ProcessName
-  | extend timestamp = StartTime, AccountCustomEntity = Account, HostCustomEntity = Computer, IPCustomEntity = IpAddress
+  | extend timestamp = StartTime
+  | extend HostName = tostring(split(Computer, ".")[0]), DomainIndex = toint(indexof(Computer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(Computer, DomainIndex + 1), Computer)
+  | extend AccountName = tostring(split(Account, "@")[0]), AccountUPNSuffix = tostring(split(Account, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
```

Comment:
Account is actually a Name + NTDomain in this case.  So you can map it, but you have to split it as follows:

| extend AccountName = tostring(split(Account, "\\")[1]), AccountNTDomain = tostring(split(Account, "\\")[0])